recovery/cmd/backup_size.go

```diff
@@ -0,0 +1,189 @@
+package cmd
+
+import (
+	"errors"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	disk "github.com/shirou/gopsutil/v3/disk"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	clusterPath         = "/var/lib/etcd/member/snap/db"
+	staticPodsPath      = "/etc/kubernetes/static-pod-resources/"
+	usrLocalPath        = "/usr/local"
+	kubeletPath         = "/var/lib/kubelet"
+	etcPath             = "/etc"
+	binPath             = "/etc/kubernetes/static-pod-resources/bin/"
+	backupSizeSafetyNet = 10.00
```

Comment:
Maybe add units in a comment, or in the var name? Also, just wondering are these variables configurable, or should these be const?

Reply:
Added